### PR TITLE
Add missing RZ pieces in HydroUtils

### PR DESCRIPTION
**Pull request by esclapez**

### CODE REVIEW

**Comment by cgilet** [comment]
Could we combine the Cartesian and  RZ versions of ComputeFluxes and ComputeDivergence? I can't think of a good reason to maintain separate versions, especially if we're passing `geom`.

**Comment by esclapez** [comment]
I need to test this on GPU even though 2D+GPU is rarely a priority.

**Comment by esclapez** [comment]
@cgilet successfully tested on GPU.

File: Utils/hydro_utils.cpp [hunk 1]
```diff
@@ -21,65 +21,105 @@ HydroUtils::ComputeFluxes ( Box const& bx,
                             Geometry const& geom, const int ncomp,
                             const bool fluxes_are_area_weighted )
 {
+#if (AMREX_SPACEDIM == 2)
+    if (geom.IsRZ()) {
+        // Need metrics when using RZ
+        Array<FArrayBox,AMREX_SPACEDIM> area;
+        Array<Elixir,AMREX_SPACEDIM> area_eli;
+        if (!fluxes_are_area_weighted) {
```
Comment (cgilet) [inline Utils/hydro_utils.cpp:29]: I don't think the logic is right here. If we want area weighted fluxes, then we need to get the area FAB.

File: Utils/hydro_utils.cpp [hunk 2]
```diff
@@ -21,65 +21,105 @@ HydroUtils::ComputeFluxes ( Box const& bx,
...
+            for (int dir = 0; dir < AMREX_SPACEDIM; ++dir)
+            {
+                const int ngrow_area = 0;
+                geom.GetFaceArea(area[dir],BoxArray(bx),0,dir,ngrow_area);
+                area_eli[dir] = area[dir].elixir();
+            }
+        }
+        const auto& ax = (fluxes_are_area_weighted) ? Array4<const Real>{}
```
Comment (cgilet) [inline Utils/hydro_utils.cpp:37]: The logic would also need to be corrected here.

**Pull request merged**